cypress/e2e/39-projects.cy.ts

```diff
@@ -356,15 +384,201 @@ describe('Projects', () => {
+				.should('not.exist');
+
+			// Move the credential from Project 1 to Project 2
+			projects.getMenuItems().first().click();
+			workflowsPage.getters.workflowCards().should('have.length', 2);
+			projects.getProjectTabCredentials().click();
+			credentialsPage.getters.credentialCards().should('have.length', 1);
+			credentialsPage.getters.credentialCardActions('Credential in Project 1').click();
+			credentialsPage.getters.credentialMoveButton().click();
+
+			projects
+				.getResourceMoveModal()
+				.should('be.visible')
+				.find('button:contains("Next")')
+				.should('be.disabled');
+			projects.getProjectMoveSelect().click();
+			getVisibleSelect()
+				.find('li')
+				.should('have.length', 1)
+				.first()
+				.should('contain.text', 'Project 2')
+				.click();
+			projects.getResourceMoveModal().find('button:contains("Next")').click();
+
+			projects
+				.getResourceMoveConfirmModal()
+				.should('be.visible')
+				.find('button:contains("Confirm")')
+				.should('be.disabled');
+
+			projects
+				.getResourceMoveConfirmModal()
+				.find('input[type="checkbox"]')
+				.first()
+				.parents('label')
+				.click();
+			projects
+				.getResourceMoveConfirmModal()
+				.find('button:contains("Confirm")')
+				.should('be.disabled');
+			projects
+				.getResourceMoveConfirmModal()
+				.find('input[type="checkbox"]')
+				.last()
+				.parents('label')
+				.click();
+			projects
+				.getResourceMoveConfirmModal()
+				.find('button:contains("Confirm")')
+				.should('not.be.disabled')
+				.click();
+			credentialsPage.getters.credentialCards().should('not.have.length');
+			projects.getMenuItems().last().click();
+			projects.getProjectTabCredentials().click();
+			credentialsPage.getters.credentialCards().should('have.length', 2);
 		});
 	});
 });
+
+function createProject(name: string) {
+	projects.getAddProjectButton().should('be.visible').click();
+	projects
+		.getProjectNameInput()
+		.should('be.visible')
+		.should('be.focused')
+		.should('have.value', 'My project')
+		.clear()
+		.type(name);
+	projects.getProjectSettingsSaveButton().click();
+}
+
+function createWorkflow(fixtureKey: string, name: string) {
+	workflowPage.getters
+		.workflowImportInput()
+		.selectFile(`cypress/fixtures/${fixtureKey}`, { force: true });
+	workflowPage.actions.setWorkflowName(name);
+	workflowPage.getters.saveButton().should('contain', 'Saved');
+	workflowPage.actions.zoomToFit();
+}
+
+function createCredential(name: string) {
+	credentialsModal.getters.newCredentialModal().should('be.visible');
+	credentialsModal.getters.newCredentialTypeSelect().should('be.visible');
+	credentialsModal.getters.newCredentialTypeOption('Notion API').click();
+	credentialsModal.getters.newCredentialTypeButton().click();
+	credentialsModal.getters.connectionParameter('Internal Integration Secret').type('1234567890');
+	credentialsModal.actions.setName(name);
+	credentialsModal.actions.save();
+	credentialsModal.actions.close();
+}
```

Comment:
Can we move these to `cypress/composables/projects.ts`?

Reply:
Sure, done